uiowa/uiowa
# P2LB reporting tool

Resolves #6735 

# How to test

```
ddev blt ds --site brand.uiowa.edu && ddev drush @brand.local config-split:activate p2lb -y && ddev drush @brand.local cim -y && ddev drush @brand.local uli node/4406/edit
```
- Observe the warning.
- Click the "V3 Status" tab and observe output. It should match up to https://github.com/uiowa/uiowa/pull/6708#issuecomment-1662863485
- Convert the page.
- See the status is updated on the "V3 Status" tab.
- See that the warning no longer shows on the "Edit" tab.
- The idea with this is not that it is complete, but that the experience of using it is good enough to merge it so we can add more too it later.
